### DISCUSSION TITLE
目标端为Windows如何才能使用？

### DISCUSSION THREAD

**Discussion by daybug** [juicedata/juicesync#100] [Q&A] [answered]
工作中有时需要帮助客户迁移文件到windows，使用rsync速度太慢，有什么办法可以使用juicesync吗？

**Comment by davies** [comment, marked as answer]
juicesync 应该可以在 Windows 上编译的，或者你用 juicefs 的 windows 版，juicefs sync 跟  juicesync 是一样的。
